packages/babel-preset-cozy-app/index.js

```diff
@@ -2,57 +2,90 @@
 
 const { declare } = require('@babel/helper-plugin-utils')
 const browserslist = require('browserslist-config-cozy')
+const { validate, isOfType } = require('./validate')
+const merge = require('lodash/merge')
 
-const browserEnv = {
+const mapValues = (obj, iter) => {
```

Comment:
why you don't use lodash [`mapValues`](https://lodash.com/docs/4.17.11#mapValues)?

Reply:
Because `babel-preset-cozy-app` is a very small module, I did not want to introduce lodash here.

Reply:
but it's already [here](https://github.com/cozy/cozy-libs/pull/321/files#diff-f61cc66dd92dad64f006e20057c023ebR6)...